king_phisher/server/database/manager.py

```diff
@@ -68,6 +68,14 @@
 _meta_data_type_map = {'int': int, 'str': str}
 _popen = lambda args: subprocess.Popen(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, close_fds=True)
 
+ALEMBIC_VERSIONS = {
```

Comment:
This information should be pulled dynamically by listing and parsing the versions directory. Something like:
```py
alembic_directory = find.find_data_directory('alembic')
versions = os.listdir(os.path.join(alembic_directory, 'versions'))
# parser versions into a dictionary possibly using a regex
```